example/spec/parallel/testAPI/granuleSpec.js

```diff
@@ -270,5 +270,85 @@ describe('The Granules API', () => {
         expect(message.deletedAt).toBeGreaterThan(timestamp);
       }
     });
+
+    it('errors creating a granule that already exists across a different collection', async () => {
+      if (beforeAllError) {
+        fail(beforeAllError);
+      }
+      const newCollection = await createCollection(prefix);
+      const newCollectionId = constructCollectionId(newCollection.name, newCollection.version);
+      const granuleToWrite = removeNilProperties(fakeGranuleFactoryV2({
+        collectionId: newCollectionId,
+        published: false,
+        dataType: undefined,
+        version: undefined,
+        execution: undefined,
```

Comment:
We may not, let me update

Reply:
`published` needs to be set to false to delete the granule, `execution` needs to be undefined (otherwise, it will try to look for the PG execution`,  Other fields are not required so let me remove those settings. Thanks, Jenny!

Reply:
https://github.com/nasa/cumulus/pull/3126/commits/b660d49a5ccfe8d1d78e4e1954d9bd2d23bfdc4d